python/ray/tests/test_runtime_env_working_dir.py

```diff
@@ -59,6 +60,12 @@ def call_ray_init():
 
     def reinit():
         ray.shutdown()
+        # Currently, reinit the driver will generate the same job id. And if we
+        # reinit immediately after shutdown, raylet may process new job started
+        # before old job finished in some cases. This inconsistency could
+        # disorder the URI reference and delete a valid runtime env. We sleep
+        # here to walk around this issue.
+        time.sleep(5)
```

Comment:
This is unfortunate, do you have any thoughts on how we could improve this in the future?

Reply:
Maybe we should use a new job id when reinit?